charts/config/values.yaml

```
@@ -48,6 +48,16 @@ runner:
     app: gitlab-runner
   namespace: gitlab-runner
   sandboxNamespace: gitlab-runner-sandbox
+
+# Gitaly is responsible for mirroring actions, and there are 4 valid protocols with the following ports
+# ssh: 22
+# http: 80
+# https: 443
+# git: 9418
+mirroring:
+  ports:
```

```suggestion
  enabled: false
  ports:
```
We should turn this off by default since many envs won't use it.

---

(also this should be added to the networking configuration section of the docs)